core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -125,24 +119,60 @@ class BillingProfileManagerDAOImpl(samDAO: SamDAO,
       )
       .flatMap {
         case true =>
-          Future.successful(
-            Seq(
-              RawlsBillingProject(
-                RawlsBillingProjectName(azureConfig.billingProjectName),
-                CreationStatuses.Ready,
-                None,
-                None,
-                azureManagedAppCoordinates = Some(
-                  AzureManagedAppCoordinates(
-                    UUID.fromString(azureConfig.azureTenantId),
-                    UUID.fromString(azureConfig.azureSubscriptionId),
-                    azureConfig.azureResourceGroupId
+          val profileApi = apiClientProvider.getProfileApi(ctx)
+          Future.successful(profileApi.listProfiles(0, 1000).getItems.asScala.toSeq)
+        case _ => Future.successful(Seq())
+      }
+  }
+
+  def getHardcodedAzureBillingProject(samUserResources: Seq[SamUserResource], userInfo: UserInfo)(implicit
```

Comment:
I have created https://broadworkbench.atlassian.net/browse/WOR-486 to delete all this code associated with the hardcoded account.